### PR TITLE
CMS: new validation records

### LINKED ISSUE #1552: Create record for the 2011 ttbar validation example
Bring to the portal https://github.com/cms-opendata-validation/2011-doubleelectron-doublemu-mueg-ttbar

@zenaiev Provide entries under the different titles for the software record (get inspiration from http://opendata.cern.ch/record/462 or http://opendata.cern.ch/record/233 but note that the header names with "data" are wrong and will be changed see #1537) 

@ArtemisLav @tiborsimko Are we happy with sw record subtitle or should we change/add/remove some? Should we unify validation with other software records?

Current visible fields are (taken into account #1537) 
If you feel that these titles do not match the contents or are not enough, we can still change them.
Title:
Authors:
Description:
Use with:
System details:
Related items:
How can you use this code:
Issues and limitations:

Note that we will give the github repository link in the record (for example in How can you use this code) but for data preservation purposes, we would also like to add the contents of the repo as a file.

### LINKED ISSUE #1556: Create record for 2011 dimuon spetrum example code
Prepare the record based on /afs/desy.de/user/g/geiser/public/CMS/opendata/MUO-10-004demo_2011/

NB We can remove from BuildFile 
```
<export>
   <lib name="1"/>
</export>
```
as instructed in the compile time warning:
****WARNING: No need to export library once you have declared your library as plugin. Please cleanup src/Demo/DemoAnalyzer/BuildFile by removing the <export></export> section.

Prepare a github repo for this under cms-opendata-analyses

### PR DESCRIPTION
**Pull request by ArtemisLav**
* Adds two new validation records (closes #1552) (closes #1556)

Signed-off-by: Artemis Lavasa <artemis.lavasa@cern.ch>

### CODE REVIEW

File: cernopendata/modules/fixtures/data/records/cms-dimuon-spectrum-2011.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    }
+  ], 
+  "collections": [
+    "CMS-Validation-Utilities"
+  ], 
+  "date_created": "2011", 
+  "date_published": "2017", 
+  "distribution": {
+    "formats": [
+      "FIXME"
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-dimuon-spectrum-2011.json]: 1 file with `gz` extension

File: cernopendata/modules/fixtures/data/records/cms-dimuon-spectrum-2011.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    ], 
+    "number_files": "FIXME", 
+    "size": "FIXME"
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-dimuon-spectrum-2011.json]: 122975 bytes

File: cernopendata/modules/fixtures/data/records/cms-dimuon-spectrum-2011.json [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  }, 
+  "doi": "10.7483/OPENDATA.CMS.D00J.UVB1", 
+  "experiment": "CMS", 
+  "files": [
+    {
+      "checksum": "FIXME", 
+      "size": "FIXME", 
+      "type": "xrootd", 
+      "uri": "FIXME"
+    }
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-dimuon-spectrum-2011.json]: - /eos/opendata/cms/software/DimuonSpectrum2011/DimuonSpectrum2011-1.0.0.tar.gz
- size 122975
- no type
- checksum sha1 4bfc8b8bad13b9c2dbf2d091cc6bf171a1877845

File: cernopendata/modules/fixtures/data/records/cms-validation-code-ttbar.json [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    }
+  ], 
+  "collections": [
+    "CMS-Validation-Utilities"
+  ], 
+  "date_created": "2011", 
+  "date_published": "2017", 
+  "distribution": {
+    "formats": [
+      "FIXME"
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-validation-code-ttbar.json]: `gz` format, 1 file, 1288913 bytes

File: cernopendata/modules/fixtures/data/records/cms-validation-code-ttbar.json [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  }, 
+  "doi": "10.7483/OPENDATA.CMS.TTGV.3IKL", 
+  "experiment": "CMS", 
+    "files": [
+    {
+      "checksum": "FIXME", 
+      "size": "FIXME", 
+      "type": "xrootd", 
+      "uri": "FIXME"
+    }
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-validation-code-ttbar.json]: - /eos/opendata/cms/software/2011-doubleelectron-doublemu-mueg-ttbar/2011-doubleelectron-doublemu-mueg-ttbar-1.0.0.tar.gz
- size 1288913
- no type
- checksum sha1 82c1d9bc3204ee8d106acab68d1aa56a1bc3e4fb

**Comment by ArtemisLav** [comment]
Thanks, changes incorporated & branch repushed.

File: cernopendata/modules/fixtures/data/records/cms-validation-code-ttbar.json [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+      "gz"
+    ],
+    "number_files": 1,
+    "size": 1288913
+  },
+  "doi": "10.7483/OPENDATA.CMS.TTGV.3IKL",
+  "experiment": "CMS",
+    "files": [
+    {
+      "checksum": "sha1 82c1d9bc3204ee8d106acab68d1aa56a1bc3e4fb",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-validation-code-ttbar.json]: `sha1:82c1d9bc3204ee8d106acab68d1aa56a1bc3e4fb` please note we need `:` in between

File: cernopendata/modules/fixtures/data/records/cms-dimuon-spectrum-2011.json [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+      "gz"
+    ],
+    "number_files": 1,
+    "size": 122975
+  },
+  "doi": "10.7483/OPENDATA.CMS.D00J.UVB1",
+  "experiment": "CMS",
+  "files": [
+    {
+      "checksum": "sha1 4bfc8b8bad13b9c2dbf2d091cc6bf171a1877845",
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-dimuon-spectrum-2011.json]: `sha1:4bfc8b8bad13b9c2dbf2d091cc6bf171a1877845` ditto here

**Pull request merged**